README.md

```diff
@@ -1,19 +1,32 @@
 # kArmor
 
-**kArmor** is a CLI client to help manage [KubeArmor](github.com/kubearmor/KubeArmor)
+**kArmor** is a CLI client to help manage [KubeArmor](github.com/kubearmor/KubeArmor).
 
 KubeArmor is a container-aware runtime security enforcement system that
 restricts the behavior (such as process execution, file access, and networking
 operation) of containers at the system level.
 
 ## Installation
 
+The following sections show how to install the kAmor. It can be installed either from source, or from pre-built binary releases.
+
+### From Script
+
+kAmor has an installer script that will automatically grab the latest version of kAmor and install it locally.
```

Comment:
```suggestion
kArmor has an installer script that will automatically grab the latest version of kArmor and install it locally.
```

s/kAmor/kArmor across the changes 😅

Reply:
Still a few `kAmor` --> `kArmor` changes needed 😅 😬

Reply:
Done